### PR TITLE
CS fixes for 2.6

### PR DESCRIPTION
**Pull request by keradus**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | ? |
| Fixed tickets | N/A |
| License | MIT |
| Doc PR | N/A |

To keep fabbot.io happy ;)

### CODE REVIEW

**Comment by keradus** [comment]
fabbot.io fails on bug reported here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1288

File: src/Symfony/Bundle/FrameworkBundle/Tests/Templating/Helper/Resources/Custom/_name_c_entry_label.html.php [hunk 1]
```diff
@@ -1,2 +1,4 @@
-<?php if (!$label) { $label = $view['form']->humanize($name); } ?>
+<?php if (!$label) {
+    $label = $view['form']->humanize($name);
+} ?>
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Templating/Helper/Resources/Custom/_name_c_entry_label.html.php]: should be reverted

File: src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php [hunk 2]
```diff
@@ -333,7 +333,7 @@ public function testNonDecoratedOutput()
             " 140/200 [===================>--------]  70%\n".
             " 160/200 [======================>-----]  80%\n".
             " 180/200 [=========================>--]  90%\n".
-            " 200/200 [============================] 100%",
+            ' 200/200 [============================] 100%',
```
Comment (fabpot) [inline src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php]: should be reverted to be consistent with the other lines

File: src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php [hunk 3]
```diff
@@ -351,7 +351,7 @@ public function testNonDecoratedOutputWithClear()
         $this->assertEquals(
             "  0/50 [>---------------------------]   0%\n".
             " 25/50 [==============>-------------]  50%\n".
-            " 50/50 [============================] 100%",
+            ' 50/50 [============================] 100%',
```
Comment (fabpot) [inline src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php]: same as above.

File: src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php [hunk 4]
```diff
@@ -365,7 +365,7 @@ public function testNonDecoratedOutputWithoutMax()
         rewind($output->getStream());
         $this->assertEquals(
             "    0 [>---------------------------]\n".
-            "    1 [->--------------------------]",
+            '    1 [->--------------------------]',
```
Comment (fabpot) [inline src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php]: same here as well

File: src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php (hunk 2 above)
Comment (keradus) [inline src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php]: Then fabbot.io will always complains about it. Is it OK?

Comment (fabpot) [inline src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php]: I won't let a bot tell me what's good :)

If we really want to make fabbot happy, we could change the previous lines to single quotes: `'...'."\n"`, but at this point, I don't really understand why we are spending hours on these CS "issues".

**Comment by fabpot** [comment]
Closing for now as there are many changes that need to be reverted.

**Pull request closed without merging**